requirements

```diff
@@ -1,2 +1,2 @@
-django>=1.4,<1.9
-ua_parser==0.7.1
\ No newline at end of file
+django==1.9
```

Comment:
We shouldn't break backwards compatibility. Should be `django>=1.4,<1.10`. Might want to jump right to 1.11?